tests/integration-service/namespace-backed-environments.go

```diff
@@ -140,10 +142,10 @@ var _ = framework.IntegrationServiceSuiteDescribe("Namespace-backed Environment
 		It("checks for spaceRequest after Ephemeral env has been created", func() {
 			Eventually(func() error {
 				sr, err = f.AsKubeAdmin.GitOpsController.GetSpaceRequests(testNamespace)
-				if sr.Items == nil {
-					return fmt.Errorf("Space request is nil.")
+				if sr.Items == nil || len(sr.Items) == 0 {
+					return fmt.Errorf("No Space request is found.")
 				}
-				if sr.Items[0].Status.Conditions[0].Type != v1alpha1.ConditionType("Ready") {
+				if sr.Items[0].Status.Conditions != nil && sr.Items[0].Status.Conditions[0].Type != v1alpha1.ConditionType("Ready") {
```

Comment:
I think we should use the same approach with `meta.IsStatusConditionTrue()` here to guard against there being more than one type of status condition in the Space CR.

Reply:
I tried, however this data type of condition differs from the other one in **gmel**, I am affraid that meta.IsStatusConditionTrue() cant be use in this case.

Reply:
Ok, now I see that it's a [custom type](https://github.com/codeready-toolchain/api/blob/master/api/v1alpha1/conditions.go). I think we can keep it as-is for now as it seems like there's only one type (`Ready`) defined for it.

There's a small worry of the possibility of new condition types getting added, but I think we can table it for now.